debian/odoo.conf

```diff
@@ -5,4 +5,4 @@ db_host = False
 db_port = False
 db_user = odoo
 db_password = False
-addons_path = /usr/lib/python2.7/dist-packages/odoo/addons
+addons_path = /usr/lib/python3/dist-packages/odoo/addons
```

Comment:
`addons_path` is not necessary.

Reply:
You are right, it's computed, I'll remove it.